js/dep/helpers.js

```diff
@@ -0,0 +1,74 @@
+var helpers = {
+  getUTCDatetime: function () {
+    var now = new Date()
+    var d = new Date(
```

Comment:
the single character variable names make some of the code hard to understand. Is it possible to leave this work to uglify instead?